files/zh-cn/glossary/decryption/index.md

```diff
@@ -5,7 +5,7 @@ slug: Glossary/Decryption
 
 {{GlossarySidebar}}
 
-在密码学（{{glossary("cryptography")}}）领域，**解密**是指把密文（{{glossary("ciphertext")}}）转换成明文（{{glossary("Plaintext")}}）的过程。
+在密码学（{{glossary("cryptography")}}）领域，**解密**是指把{{glossary("ciphertext", "密文")}}转换成{{glossary("Plaintext", "明文")}}的过程。
 
 解密是一个密码学原语：它通过一种称作{{glossary("cipher")}}的编码技术，把加密信息转换成纯文本。正如加密一样，解密在当代密码学领域也是通过特有的算法，结合称作{{glossary("key")}}的密钥来工作。由于算法通常是公开的，若要保证加密安全，就必须确保密钥高度保密。
 
```

Comment:
Please translate the image description and the see also section below.

Reply:
link in the seealso section has a problem. will have a fix in the upstrream source.
<img width="380" alt="Screenshot 2024-05-09 at 7 14 44 PM" src="https://github.com/mdn/translated-content/assets/41184947/20708508-b314-4c11-95e7-decbb9a67b30">